test/python/transpiler/test_preset_passmanagers.py

```diff
@@ -279,7 +280,7 @@ def counting_callback_func(pass_, dag, time, property_set, count):
             callback=counting_callback_func,
             translation_method="synthesis",
         )
-        self.assertEqual(gates_in_basis_true_count + 1, collect_2q_blocks_count)
+        self.assertEqual(gates_in_basis_true_count + 2, collect_2q_blocks_count)
```

Comment:
If anyone else is looking at this, this change is needed because with this PR applied we're calling `Collect2qBlocks` an extra time during the `init` stage now (it took me a sec to realize why this changed).